tests/test_marathon.py

```diff
@@ -582,6 +677,10 @@ def _create_fixtures():
     return marathon_client, rpc_client
 
 
+def _default_if_none(value, default):
```

Comment:
Could you clarify this comment? Did you mean "inline this method"? Or, "put the default values as default arguments in the test method that calls this"? Or, "don't use default arguments at all, and specify them explicitly for every test case"?